java/ql/test/library-tests/locations/WildcardLocations.ql

```diff
@@ -1,4 +1,30 @@
 import default
+import TestUtilities.InlineExpectationsTest
 
-from WildcardTypeAccess wta
-select wta
+private string getValue(WildcardTypeAccess wta) {
+  if wta.hasNoBound() // also makes sure that hasNoBound() is working correctly
+  then result = ""
+  else
+    // Also makes sure that getBound() is working correctly (and has at most
+    // one result)
+    exists(Expr bound | bound = wta.getBound() and not wta.getBound() != bound |
```

Comment:
```suggestion
    exists(Expr bound | bound = wta.getBound() |
```
It's better to test the not-having-multiple-values by potentially observing those values rather than observing nothing.

Reply:
Though then the test would also succeed when `getBound()` would have completely unrelated results since here in the test its result is bound below with `getUpperBound()` / `getLowerBound()`.